src/components/ui/Input.tsx

```diff
@@ -82,7 +82,7 @@ const Input = forwardRef<HTMLInputElement, InputProps>(function (
   );
 
   return (
-    <div className="relative">
+    <div data-testid="input" className="relative">
```

Comment:
we might need to make these `data-testid` as props just so in case we are testing multiple input field in another component we can easily target them